packages/flex-plugin-webpack/src/devServer/pluginServer.ts

```diff
@@ -167,6 +198,10 @@ export const _startServer = (
 
     const hasRemotePlugin = config.remoteAll || plugins.remote.length !== 0;
     const localPlugins = _getLocalPlugins(config.port, plugins.local);
+    let versionPlugins: Plugin[] = [];
```

Comment:
`const` and then later `versionPlugins.concat(_getRemoteVersionPlugins(plugins.version))`

Reply:
When I make this singular change the functionality breaks so I've kept it the way it was before